### PR TITLE
:technologist: Sort imports automatically

### LINKED ISSUE #6: [Feature] Sort imports using @trivago/prettier-plugin-sort-imports
### Overview

Sort imports using [@trivago/prettier-plugin-sort-imports](https://www.npmjs.com/package/@trivago/prettier-plugin-sort-imports/v/2.0.0).

### Describe the solution you'd like

Automatically sort imports on save.

### PR DESCRIPTION
**Pull request by injoonH**
Closes #6

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -10,5 +10,9 @@
   "bracketSpacing": true,
   "bracketSameLine": false,
   "arrowParens": "always",
-  "endOfLine": "lf"
+  "endOfLine": "lf",
+  "importOrder": ["^([@]?)([a-zA-Z]+)([/]?)(.*)$", "^@/(.*)$", "^../(.*)", "^./(.*)"],
+  "importOrderSeparation": true,
+  "importOrderSortSpecifiers": true,
```
Comment (yumincho) [inline .prettierrc:16]: `importOrderSortSpecifiers`는 뭔가요?

Comment (injoonH) [reply .prettierrc:16]: [@trivago/prettier-plugin-sort-imports#importordersortspecifiers](https://www.npmjs.com/package/@trivago/prettier-plugin-sort-imports#importordersortspecifiers)

> A boolean value to enable or disable sorting of the specifiers in an import declarations.

Comment (yumincho) [reply .prettierrc:16]: `importOrder` 이해했습니당 👍  (외부 라이브러리 ➡️ 최상위 디렉토리 ➡️ 상위 디렉토리 ➡️  현재 디렉토리)

**Review by yumincho — APPROVED** [review]
👍

**Pull request merged**